spreg/sur.py

```diff
@@ -312,7 +316,7 @@ class SUR(BaseSUR, REGI.Regimes_Frame):
     (bigXvars). All these will be created from th database (db) and lists
     of variables (y_var and x_var) created above.
 
-    >>> bigy,bigX,bigyvars,bigXvars = pysal.spreg.sur_utils.sur_dictxy(db,y_var,x_var)
+    >>> bigy,bigX,bigyvars,bigXvars = sur_dictxy(db,y_var,x_var)
```

Comment:
Should this be formatted with spaces as:
```python
bigy, bigX, bigyvars, bigXvars = ...
```

Reply:
I am not dealing with stylish formatting in this PR since it would confuse the changes for dealing with the libpysal example API changes (- make the PR difficult to read).